srcCpp/RTIDDSImpl.h

```diff
@@ -125,11 +87,21 @@ class RTIDDSImpl : public IMessaging
     bool configureDomainParticipantQos(DDS_DomainParticipantQos &qos);
 
   #ifndef RTI_MICRO
+
+    /**
+     * @brief This function calculate the overhead bytes that all the
+     * members on TestData_* type add excluding the length of the sequence.
+     *
+     * @param size \b InOut. The size of the overhead of the data type.
+     *
+     * @return true if the operation was successful, otherwise false.
+     */
+    virtual bool get_serialize_overhead_size(unsigned int &overhead_size);
```

Comment:
I think this would be "get_serialized_overhead_size()"

Reply:
DONE